cypress/integration/logistics/create_delivery_days.js

```diff
@@ -0,0 +1,67 @@
+import { BPartner } from '../../support/utils/bpartner';
+import { toggleNotFrequentFilters, selectNotFrequentFilterWidget, applyFilters } from '../../support/functions';
+import { humanReadableNow } from '../../support/utils/utils';
+import { Tour } from '../../support/utils/tour';
+import { TourVersion } from '../../support/utils/tour_version';
+
+describe('Create a tour and a tourversion', function() {
+  const date = humanReadableNow();
+  const customerName = `Customer ${date}`;
+  const tourName = `TestTour ${date}`;
+  const tourVersionName = `TestTourVersion ${date}`;
+  // let mainTourId;
+
+  it('Create customer for which we will create a tour', function() {
+    cy.fixture('sales/simple_customer.json').then(customerJson => {
+      new BPartner({ ...customerJson, name: customerName }).setCustomer(true).apply();
+    });
+  });
+  it('Create a tour', function() {
+    cy.fixture('logistics/tour.json').then(tourJson => {
+      Object.assign(new Tour(), tourJson)
+        .setName(tourName)
+        .apply();
+    });
+    // cy.get(`@${tourName}`).then(mainTour => {
+    //   mainTourId = mainTour.documentId;
+    // });
+  });
+  it('Create a tour version', function() {
+    cy.fixture('logistics/tourversion.json').then(tourVersionJson => {
+      Object.assign(new TourVersion(), tourVersionJson)
+        .setName(tourVersionName)
+        .setTour(tourName)
+        .setCustomer(customerName)
+        .apply();
+    });
+  });
+  it('Create delivery days for this tour', function() {
+    cy.visitWindow(540331);
+    toggleNotFrequentFilters();
+    selectNotFrequentFilterWidget('default');
+    cy.writeIntoStringField('Name', tourName, false, null, true);
+    applyFilters();
```

Comment:
I have tried to navigate directly to that tour instead of filtering, a similar solution to what we have in create_bill_of_material.js 

cy.get(`@${productName}`).then(mainProduct => {
      mainProductId = mainProduct.documentId;
    });
and then
cy.visitWindow('140', mainProductId); //in my case another windowId - but I couldn't get it to work properly. Any help would be appreciated, or we can just leave it like this.